public/index.css

```diff
@@ -10,6 +10,17 @@ body {
 }
```

Comment:
You put all of your styles in the completely wrong file. Please, for all component create a separate directory with 2 files: index and styles. And put corresponding styles to where they belong.